lib/src/rules/prefer_final_locals.dart

```diff
@@ -46,6 +46,10 @@ void mutableCase() {
 ''';
 
 class PreferFinalLocals extends LintRule {
+  static const LintCode code = LintCode(
+      'prefer_final_locals', 'Local variables should be final.',
+      correctionMessage: 'Try making the variables final.');
```

Comment:
I think this is reported on each variable, in which case maybe 

    'Try making the variable final.'

Reply:
Done.